build-system/tasks/performance/index.js

```diff
@@ -28,11 +29,12 @@ const {printReport} = require('./print-report');
  */
 async function performance() {
   installPackages(__dirname);
-  const {headless, runs, urls} = new loadConfig();
+  const {headless, runs, urls, handlers} = new loadConfig();
   await cacheDocuments(urls);
   await compileScripts(urls);
   await rewriteScriptTags(urls);
-  await getMetrics(urls, {headless, runs});
+  await rewriteAnalyticsTags(urls);
```

Comment:
I think `rewriteScriptTags` and `rewriteAnalyticsTags` can be merged to one to avoid read and write files multiple times. 
Another optimization would be to not invoke `rewriteAnalyticsTags` for urls without the analytics handler defined. 
Not blocking, but may be a good optimization to have once we start to run the task on travis. cc @estherkim

Reply:
> Another optimization would be to not invoke rewriteAnalyticsTags for urls without the analytics handler defined.

Sounds good to me.

> I think rewriteScriptTags and rewriteAnalyticsTags can be merged to one to avoid read and write files multiple times. 

Maybe once the ads team checks in their handler, we can see how we can group the rewrites (assuming that they need to rewrite something).